metrics/metrics/src/main/java/io/helidon/metrics/HelidonMetric.java

```diff
@@ -55,8 +55,9 @@ interface HelidonMetric extends io.helidon.metrics.api.HelidonMetric {
      * @param sb the {@code StringBuilder} used to accumulate the output
      * @param metricID the {@code MetricID} for the metric to be formatted
      * @param withHelpType flag to control if TYPE and HELP are to be included
+     * @param isStrictExemplars whether strict exemplar handling is in force
      */
-    void prometheusData(StringBuilder sb, MetricID metricID, boolean withHelpType);
+    void prometheusData(StringBuilder sb, MetricID metricID, boolean withHelpType, boolean isStrictExemplars);
 
     /**
      * Return a name for this metric, possibly including a unit suffix.
```

Comment:
Isn't this a breaking API change for dot release? Could we do both method signatures to provide backward compatibility?

Reply:
This is not a breaking change; `HelidonMetric` is not a public class.

Reply:
My bad sorry :facepalm: